lib/diagnostic.rb

```diff
@@ -20,6 +23,8 @@ def sum(enum)
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
+    enum.each_with_object(false) { |o, m| m = true if o.odd? }
+    m
```

Comment:
Almost! `m` doesn't exist outside of the block above, so having that be the last line causes the method to return `undefined`. If you had just left that line out, it would have worked, since `.each_with_object` returns the modified object as a result.